src/Microsoft.AspNet.Mvc.Razor/Razor/PreCompileViews/RazorPreCompiler.cs

```diff
@@ -15,39 +18,42 @@ namespace Microsoft.AspNet.Mvc.Razor
 {
     public class RazorPreCompiler
     {
+        private static readonly int MaxDegreesOfParallelism = Environment.ProcessorCount;
```

Comment:
in what way is parallel compilation an implementation detail?  e.g. see `-maxthreads` options on many top-level command lines.  further users / administrators may wish to limit each site to a single thread.  so :+1: to @yishaigalatzer's suggestion.